### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by gitshanks**
Added tests for Profile POST and GET with MongoDB

### CODE REVIEW

File: python-app/test_profile.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+mydb = myclient["STUDYBUDDY"]
+mycollections = mydb["user_details"]
+
+data["name"] = "Test_Name"
+data["skills"] = "Test_Skills"
+data["courses"] = "Test_Courses"
+data["department"] = "Test_Department"
+
+
+def test_post_profile():
```
Comment (yasa0041) [inline python-app/test_profile.py:19]: use a test collection and don't test with the production collection.

Also, delete your collection after all the test cases have ran.

File: python-app/main.py [hunk 2]
```diff
@@ -62,18 +68,48 @@ def create_post():
...
+    req = request.json
+    data = {}
+    myclient = pymongo.MongoClient(
+        "mongodb+srv://admin:admin@cluster0-jacon.gcp.mongodb.net/test?retryWrites=true&w=majority")
+    mydb = myclient["STUDYBUDDY"]
+    mycollections = mydb["user_details"]
+    data["name"] = req["name"]
+    data["skills"] = req["skills"]
+    data["courses"] = req["courses"]
+    data["department"] = req["department"]
```
Comment (yasa0041) [inline python-app/main.py:108]: where is the part to store the profile image?
You'll need to store the image on the server and persist the path in Mongo collection

File: python-app/test_profile.py (hunk 1 above)
Comment (gitshanks) [reply python-app/test_profile.py:19]: It is a test collection. Please check

File: python-app/main.py (hunk 2 above)
Comment (gitshanks) [reply python-app/main.py:108]: Image will be covered in later stories

File: python-app/test_profile.py (hunk 1 above)
Comment (yasa0041) [reply python-app/test_profile.py:19]: delete this new collection after test run, otherwise  there'll always be an entry for name='Test_Name'

**Pull request merged**